<launch>

  <node pkg="vision" type="cascade_classifier" name="detector_node" ns="aero/lower_stereo" output="screen">

    <param name="cascade_path" value="$(find vision)/trainingSnapshots/haartraining_1000samples_data.xml"/>
    <param name="show_windows" value="false"/>
  </node>

</launch>
